assets/stylesheets/_index.sass

```diff
@@ -37,6 +37,11 @@ a
   color: $text-color-dark
   padding: 50px
 
+  .sorting
```

Comment:
Hahaha. Ok, I'll fix it. (not that this style is great, I might have to make it go through a designer, eventually).